LISEZ-MOI.md

```diff
@@ -0,0 +1,30 @@
+# Marcel, docker à la française
+
+![logo](https://brouberol.github.io/marcel/images/logo/marcel-logo.png)
+
+Marcel est une enveloppe française qui se base sur l'interface de commande docker et a pour but de remplacer docker, afin de préparer le chemin pour le nouvel Système d'Exploitation souverain Français.
```

Comment:
I'm currently performing a  refactoring to integrate unit tests. Please
wait until I've pushed to translate the README.

--
B
 
 
On Sun, Jan 24, 2016, at 12:02 PM, Arnaud Ligny wrote:
> In LISEZ-MOI.md[1]:


> > @@ -0,0 +1,30 @@
> +# Marcel, docker à la française
> +
> +![logo](https://brouberol.github.io/marcel/images/logo/marcel-logo.png)
> +
> +Marcel est une enveloppe française qui se base sur l'interface de commande docker et a pour but de remplacer docker, afin de préparer le chemin pour le nouvel Système d'Exploitation souverain Français.
>  * "surcouche" plutôt "enveloppe" ?
>  * "et a pour but" => "et qui a pour but"
>  * "pour le nouvel" => "pour le nouveau"
> —
> Reply to this email directly or view it on GitHub[2]. 


>  
 


Links:

  1. https://github.com/brouberol/marcel/pull/30#discussion_r50633809
  2. https://github.com/brouberol/marcel/pull/30/files#r50633809